projects/eventonica/README.md

```diff
@@ -15,7 +15,7 @@ You'll work on this project over the next 5 weeks. As you learn more web develop
 1. [Part 4 - API's](./eventonica-part4-apis.md)
 1. [Part 5 - Express Backend](./eventonica-part5-express-backend.md)
 1. [Part 6 - Postgres Database](./eventonica-part6-postgres.md)
-
+1. [Part 7 - React App](./eventonica-part7-react.md)
 ### High Level Overview
```

Comment:
Add a new line after the line you added.

Reply:
What exactly? Like just a br tag?

Reply:
It is done. Please give it a review.

Reply:
The new line is not appearing yet.  Did you push your change to the branch?